Formula/grpc.rb

```diff
@@ -75,7 +76,8 @@ def install
         return GRPC_STATUS_OK;
       }
     EOS
-    system ENV.cc, "test.cpp", "-I#{include}", "-L#{lib}", "-lgrpc", "-o", "test"
+    pkg_config_flags = `pkg-config --cflags --libs protobuf grpc++`.chomp.split
```

Comment:
This is a little better readable
```suggestion
    pkg_config_flags = shell_output("pkg-config --cflags --libs protobuf grpc++").chomp.split
```

Reply:
Good catch, thanks. Applied.